### DISCUSSION TITLE
Tokens and Shippings

### DISCUSSION THREAD

**Discussion by KristopherMackowiak** [abuzuhri/Amazon-SP-API-CSharp#4] [Q&A]
Are you planning to add Token for PII and Shipping support  soon?

**Comment by abuzuhri** [comment]
> Are you planning to add Token for PII and Shipping support  soon?

Yes many changes we added soon like token for PII , token cache, shipping support

**Comment by abuzuhri** [comment]
@KristopherMackowiak shipping now added

**Reply by KristopherMackowiak** [reply]
You are great! I'm waiting for the Token for PII :)

**Reply by abuzuhri** [reply]
> You are great! I'm waiting for the Token for PII :)

Soooon

**Reply by abuzuhri** [reply]
> You are great! I'm waiting for the Token for PII :)
@KristopherMackowiak
Please contact me I'm trying to finalize PII but I need application have access to PII for test , tareq.abuzuhri@gmail.com

**Reply by abuzuhri** [reply]
> You are great! I'm waiting for the Token for PII :)

Now PII Token ready but need test, I don't have developer application approved for PII

**Reply by KristopherMackowiak** [reply]
Hi,
I'm struggling to have access to PII back. As soon as I manage to do this, I will let you know!

**Reply by abuzuhri** [reply]
Thanks

**Comment by abuzuhri** [comment]
Hi @KristopherMackowiak 

any update for PII ?

**Reply by feetpeoplellc** [reply]
Hi Abuzuhri,

I am in need of the PII report, "Unshipped Orders" (ReportType = GET_FLAT_FILE_ACTIONABLE_ORDER_DATA_SHIPPING) and have been granted access to PII.  Therefore, if you need someone to test, I volunteer!
